server/broadcast_test.go

```diff
@@ -704,7 +733,9 @@ func TestTranscodeSegment_CompleteSession(t *testing.T) {
 	// Create stub server
 	ts, mux := stubTLSServer()
 	defer ts.Close()
+	transcodeDelay := 100 * time.Millisecond
```

Comment:
I'd try to make unit tests time-dependent, because the test is inherently flaky. Especially that 100ms is very low time, so this test may fail in many environments.